test/bootstrap.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+require 'test/PhpUnitCompat.php';
```

Comment:
```suggestion
require __DIR__ . '/test/PhpUnitCompat.php';
```

Reply:
Okay. I'll fix it with the next force-push.